### ISSUE TITLE
The mininfied version does not work in Internet Explorer

### ISSUE THREAD

**Issue by calin1611** [mholt/PapaParse#565]
When using the minified version I receive this error in the console:
![image](https://user-images.githubusercontent.com/16622833/45208916-6689a200-b294-11e8-9c84-5d8a0fe0036a.png)

The file starts with this multi-line comment:
```
/*@license
	Papa Parse
	v4.6.0
	https://github.com/mholt/PapaParse
	License: MIT
*/
```

Internet Explorer being Internet Explorer doesn't know how to interpret a multi-line comment. I managed to fix this by just adding a **space** or a **new line** between `/* ` and `@license` in `papaparse.min.js`.

**Comment by dboskovic** [comment]
I've definitely seen this issue crop up before from other libraries. Contribution welcome.

**Comment by dboskovic** [comment]
@calin1611 - just to make it easier to replicate - can you share the IE version and a JS Fiddle that has the problem?

**Comment by pokoli** [comment]
I've included this fix on the 4.6.1 release so it should work now on Internet Explorer also

**Comment by dbelling** [comment]
Our team is using `v4.6.3` of this library, and still seem to be having IE compatibility issues. If this was fixed, there seems to be a regression in this particular sem ver.
